### PR TITLE
Re-factored NM-68: Refactoring NM-9

### PR DESCRIPTION
**Pull request by HomerJS**
1. Delete russian comments
2. Add  comments 
3. Use code-style
4. Delete empty class
5. Delete config with empty class

### CODE REVIEW

File: app/code/local/BeC/Notify/Model/Resource/Requests.php [hunk 1]
```diff
@@ -1,8 +1,21 @@
 <?php
```
Comment (aandolg) [inline app/code/local/BeC/Notify/Model/Resource/Requests.php]: Why is empty line?

File: app/code/local/BeC/Notify/Model/Resource/Requests/Collection.php [hunk 2]
```diff
@@ -1,8 +1,22 @@
 <?php
+
```
Comment (aandolg) [inline app/code/local/BeC/Notify/Model/Resource/Requests/Collection.php]: Why is empty line?

File: app/code/local/BeC/Notify/etc/config.xml [hunk 3]
```diff
@@ -21,53 +19,13 @@
...
         <resources>
-            <notify_setup>
+            <bec_notify_setup>
                 <setup>
                     <module>BeC_Notify</module>
                 </setup>
-            </notify_setup>
+            </bec_notify_setup>
         </resources>
-
```
Comment (aandolg) [inline app/code/local/BeC/Notify/etc/config.xml]: Why removed the default helper? It is used for localization.

File: app/code/local/BeC/Notify/Model/Resource/Requests.php (hunk 1 above)
Comment (HomerJS) [inline app/code/local/BeC/Notify/Model/Resource/Requests.php]: Done

File: app/code/local/BeC/Notify/Model/Resource/Requests/Collection.php (hunk 2 above)
Comment (HomerJS) [inline app/code/local/BeC/Notify/Model/Resource/Requests/Collection.php]: Done

File: app/code/local/BeC/Notify/etc/config.xml (hunk 3 above)
Comment (HomerJS) [inline app/code/local/BeC/Notify/etc/config.xml]: Add helper config

**Comment by HomerJS** [comment]
- Add localization file
- Add config to localization

**Pull request merged**